source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/EffluxType.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Crown copyright
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.shared.domain.v2.characteristics.adms;
+
+/**
+ * Efflux type determines
+ */
+public enum EffluxType {
+  VELOCITY(0, true, true, false, true, false, false),
+  VOLUME(1, true, false, true, true, false, false),
+  MOMENTUM(2, false, false, false, false, true, false),
+  MASS(3, true, false, false, true, false, true);
+
+  private final boolean actualOrNtp;
+  private final boolean velocity;
+  private final boolean volumeFlux;
+  private final boolean rho;
+  private final boolean fmbm;
+  private final boolean massFlux;
+  private final int type;
+
+  EffluxType(final int type, final boolean actualOrNtp, final boolean velocity, final boolean volumeFlux, final boolean rho, final boolean fmbm,
+      final boolean massFlux) {
+    this.type = type;
+    this.actualOrNtp = actualOrNtp;
+    this.velocity = velocity;
+    this.volumeFlux = volumeFlux;
+    this.rho = rho;
+    this.fmbm = fmbm;
+    this.massFlux = massFlux;
+  }
```

Comment:
Not sure what this enum determines or what each boolean represents (would guess expected fields?), but is this related to output velocity? Might be we're only supporting velocity with actual outflow velocity values being specified if I understood last wednesdays discussion right.

Also, is efflux the same as outflow?

Reply:
I've updated the comment. This enum specifies which of the fields are available for the specific efflux type. This is not yet used, but intended for among others the ui to guide what input fields to display.